devops/prettier-config/index.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  singleQuote: true,
+  arrowParens: 'avoid',
```

Comment:
This one is a matter of personal preference (I prefer no parens in JS), however in TypeScript you are forced to add them back since you can't type an arrow function without parenthesis:

```javascript
// This won't work
let addOne = n:number => n + 1

let addOne = (n: number) => n + 1
```

Reply:
I agree with Francesco here, I prefer no parens in JS as well, but with Typescript I believe it's more convenient to have them since it's easier to add types later on. That's also why Prettier changed default value to `always` with 2.0 release, some reasoning here: https://prettier.io/docs/en/options.html#arrow-function-parentheses

Reply:
Very reasonable argument, I will keep the entry and update it to 'always' to be explicit :+1:

Reply:
fixed in 60e74961a5973da577257a64532946aa4830bde9